packages/cli-repl/src/mongosh-repl.ts

```diff
@@ -248,6 +252,7 @@ class MongoshNodeRepl implements EvaluationListener {
     this.lineByLineInput.start();
     repl.setPrompt(await this.getShellPrompt(internalState));
     repl.displayPrompt();
```

Comment:
I think i miss lots of context here, but isn't this supposed to just `return` here since this.started is set on the line after?

Reply:
Ok, thanks for catching this, this should work now :+1: